src/configs.rs

```diff
@@ -47,3 +62,41 @@ pub const FIXED_SEED: [u8; 32] = [
     89, 15, 245, 34, 234, 120, 17, 218, 167, 20, 216, 9, 59, 62, 123, 217, 29, 137, 138, 115, 62,
     152, 136, 135, 48, 127, 151, 205, 40, 7, 51, 131,
 ];
+
+pub fn on_string_config_set(
+    config_ctx: &ConfigurationContext,
+    name: &str,
+    val: &'static ValkeyGILGuard<ValkeyString>,
+) {
+    let v = val.get(config_ctx);
+    let value_str = v.to_string_lossy();
+
+    let value = match value_str.parse::<f64>() {
+        Ok(v) => v,
+        Err(_) => {
+            let _: Result<(), ValkeyError> = Err(ValkeyError::Str("Invalid floating-point value"));
+            return;
```

Comment:
We need to return an error...

```suggestion
            return Err(ValkeyError::Str("Invalid floating-point value"));
```